packages/terra-form-radio/src/Radio.scss

```diff
@@ -45,90 +45,82 @@
 
     /* Custom radio shown when theme uses custom styling */
     .native-input + .outer-ring {
-      background-color: var(--terra-form-radio-outer-ring-background-color, none);
-      border-color: var(--terra-form-radio-outer-ring-border-color, none);
-      border-radius: var(--terra-form-radio-outer-ring-border-radius, none);
-      border-style: var(--terra-form-radio-outer-ring-border-style, none);
-      border-width: var(--terra-form-radio-outer-ring-border-width, none);
+      background-color: var(--terra-form-radio-outer-ring-background-color, transparent);
+      border: var(--terra-form-radio-outer-ring-border, 0);
+      border-radius: var(--terra-form-radio-outer-ring-border-radius, 0);
       box-sizing: border-box;
       cursor: pointer;
       display: inline-block;
-      height: var(--terra-form-radio-outer-ring-height, none);
+      height: var(--terra-form-radio-outer-ring-height, 0);
```

Comment:
Do you want to change this height as well as the rest of the heights under native inputs to be blank instead of 0?

Reply:
This is styling a span (.outer-ring) which is a sibling to the native input element. We could set this properties as CSS custom properties without fallback values like we did for the native input styles or keep as is. What do you think? I'm fine with either way.

Reply:
Talked with @JakeLaCombe offline. Gonna leave this as is.